storage/memorystore.go

```diff
@@ -1,50 +1,46 @@
-package store
+package storage
 
 import (
 	"crypto/sha256"
-	"fmt"
 
 	"github.com/docker/notary"
-	"github.com/docker/notary/tuf/data"
 	"github.com/docker/notary/tuf/utils"
 )
 
 // NewMemoryStore returns a MetadataStore that operates entirely in memory.
 // Very useful for testing
-func NewMemoryStore(meta map[string][]byte) *MemoryStore {
+func NewMemoryStore(initial map[string][]byte) *MemoryStore {
 	var consistent = make(map[string][]byte)
-	if meta == nil {
-		meta = make(map[string][]byte)
+	if initial == nil {
+		initial = make(map[string][]byte)
 	} else {
 		// add all seed meta to consistent
-		for name, data := range meta {
+		for name, data := range initial {
 			checksum := sha256.Sum256(data)
 			path := utils.ConsistentName(name, checksum[:])
 			consistent[path] = data
 		}
 	}
 	return &MemoryStore{
-		meta:       meta,
+		data:       initial,
 		consistent: consistent,
-		keys:       make(map[string][]data.PrivateKey),
 	}
 }
 
 // MemoryStore implements a mock RemoteStore entirely in memory.
 // For testing purposes only.
 type MemoryStore struct {
-	meta       map[string][]byte
+	data       map[string][]byte
 	consistent map[string][]byte
-	keys       map[string][]data.PrivateKey
 }
 
-// GetMeta returns up to size bytes of data references by name.
+// GetSized returns up to size bytes of data references by name.
 // If size is "NoSizeLimit", this corresponds to "infinite," but we cut off at a
 // predefined threshold "notary.MaxDownloadSize", as we will always know the
 // size for everything but a timestamp and sometimes a root,
 // neither of which should be exceptionally large
-func (m *MemoryStore) GetMeta(name string, size int64) ([]byte, error) {
-	d, ok := m.meta[name]
+func (m MemoryStore) GetSized(name string, size int64) ([]byte, error) {
+	d, ok := m.data[name]
```

Comment:
Similarly to `FilesystemStore`, do we want to just call `Get` here and then do sizing afterward?  It looks like we don't check for `NoSizeLimit` when the name is a consistent name also.